### DISCUSSION TITLE
Doesn't work on AC; works fine connected via USB

### DISCUSSION THREAD

**Discussion by tfcarlin** [JGAguado/Smart_Powermeter#30] [Q&A] [answered]
What's the best way to figure out what might be going on if the board doesn't work with a 120 VAC connection but works just fine when connected via USB?

**Comment by JGAguado** [comment]
This could mean that the AC-DC power supply is damaged ( it's rated to 100-240VAC input range). Not common, but it could happen that the pins got damaged on the transporting/soldering process. 
If it's an option (meaning that you know how to do it in a safe way and you have experience), I would ask you to make sure that this is the case: measuring the AC voltage in the input, to make sure it's correctly receiving the 120VAC, and the measurement of the output DC out of the power module. 
In any case, please send me a message to jon-garcia@hotmail.com with your order number from Elecrow 😉

**Comment by tfcarlin** [comment, marked as answer]
Hum, it appears to be working correctly. Not sure what went wrong or what might have been change between USB and 120 VAC. 

Where is the best spot on the PCB to measure the DC output of the power module?

THANKS!

**Reply by JGAguado** [reply]
No problem! Glad to know that it's working fine!

Best spot would be on the Serial Interface pins holes (5V and GND)

Best regards!
